package.json

```diff
@@ -20,9 +20,12 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-builder-plugin-stache#readme",
   "dependencies": {
+    "@blackbaud/skyux-logger": "^1.0.2",
     "cheerio": "0.22.0",
+    "cross-spawn": "^6.0.5",
     "fs-extra": "3.0.1",
     "glob": "7.1.2",
+    "latest-version": "^4.0.0",
```

Comment:
I would strongly suggest not using ranges for your dependencies (`^`). This could cause difficult to location problems if one of these packages accidentally released a breaking change.

Reply:
good catch